project/whitesource.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.3")
```

Comment:
There's no reason really.
In my original implementation, when `sbt-whitesource` was still experimental (not OSS) that file was in my `.gitignore`.